subsys/metrics/src/main/java/org/commonjava/indy/metrics/reporter/ReporterIntializer.java

```diff
@@ -30,14 +40,24 @@ public void initReporter( MetricRegistry metrics ) throws Exception
             initConsoleReporter( metrics, config );
             return;
         }
-        String reporter = config.getReporter();
-        if ( IndyMetricsConfig.INDY_METRICS_REPORTER_GRPHITEREPORTER.equals( reporter ) )
+        if ( this.isExistReporter( IndyMetricsConfig.INDY_METRICS_REPORTER_GRPHITEREPORTER ) )
         {
             initGraphiteReporterForSimpleMetric( metrics, config );
             initGraphiteReporterForJVMMetric( metrics, config );
             initGraphiteReporterForHealthCheckMetric( metrics, config );
         }
 
+        if ( this.isExistReporter( IndyMetricsConfig.INDY_METRICS_REPORTER_ZABBIXREPORTER ) )
+        {
+            this.initZabbixReporterForHealthCheckMetric( metrics, config );
+            this.initZabbixReporterForJVMMetric( metrics, config );
+            this.initZabbixReporterForSimpleMetric( metrics, config );
+        }
+
+        if ( this.isExistReporter( IndyMetricsConfig.INDY_METRICS_REPORTER_CONSOLEREPORTER ) )
```

Comment:
We need a configuration example for this reporter in the .conf files...commented out like the other ones as I mentioned above. I didn't realize this was something we could enable/disable until I saw it here in the initializer...